aptos-move/aptos-gas/src/gas_meter.rs

```diff
@@ -258,6 +263,17 @@ impl GasMeter for AptosGasMeter {
         _ty_args: impl ExactSizeIterator<Item = impl TypeView>,
         args: impl ExactSizeIterator<Item = impl ValueView>,
     ) -> PartialVMResult<()> {
+        // TODO(Gas): The table extension maintains its own memory space and currently it's hard
+        //            for us to track when values are created or dropped there.
+        //            Therefore as a temporary hack, we do not consider the memory released when
+        //            values enter the table module, "leaking them" conceptually.
+        //            This special handling should be removed once we build proper memory tracking
+        //            into the table extension itself.
+        if self.is_call_table {
+            self.is_call_table = false;
```

Comment:
I'm confused, this doesn't seem to do anything?

Reply:
Yeah I can get this removed